features/show_membership_application.feature

```diff
@@ -3,21 +3,27 @@ Feature: As an Admin
   I need to review a Membership Application that has been submitted
   PT: https://www.pivotaltracker.com/story/show/133950343
 
-Background:
-Given the following applications exist:
-  | company_name | company_number | contact_person | phone_number | company_email |
-  | Hunderiet    | 1234567890     | Emma Svensson  | 1234-234567  | min@mail.se   |
-  | DoggieZone   | 2345678901     | Pam Andersson  | 0234-234567  | din@mail.se   |
-  | Tassa-in AB  | 1234367890     | Anna Knutsson  | 1234-234569  | sin@mail.se   |
+  Background:
```

Comment:
`"Given the following user exists..."`  hasn't been implemented yet.  #13 was about the membership application, not users & devise.

But given your comment in the overall conversation about this,  I understand now.  (or at least I'm less confused :-) )